python/nammu/test/test_nammu.py

```diff
@@ -618,3 +619,12 @@ def test_config_dict(self, monkeypatch):
         assert conf['version'] == '0.2'
         with pytest.raises(NammuException):
             conf['foo']
+
+    def test_save_config_dict(self, tmpdir):
+        config_file = str(tmpdir.join('settings.yaml'))
+        config = {'nammu': 1, 'author': "anu-belsunu"}
+        config_dict = ConfigDict(config_file, config)
+        save_yaml_config(config_dict, config_file)
+        with open(config_file) as io:
+            new_config_dict = yaml.safe_load(io)
```

Comment:
Worth mentioning that we can't use `get_yaml_config` here because it messes up the path, which is why we're doing the plain load.